addons/web/static/src/js/core/ajax.js

```diff
@@ -246,6 +246,9 @@ function get_file(options) {
         _.each(options.data || {}, function (v, k) {
             data.append(k, v);
         });
+        if (options.context) {
+            data.append('context', JSON.stringify(options.context));
```

Comment:
should not be needed, thanks to the change here: https://github.com/odoo/odoo/commit/2cc360b61889e4c2831c815e1f4358bcd75ccff3#diff-2c418cb78b7c562a79e7032645a537d2R876